org.kde.WaylandDecoration.QAdwaitaDecorations.json

```diff
@@ -27,7 +26,6 @@
             "build-commands": [
                 "mkdir -p ${FLATPAK_DEST}/share/appdata",
                 "cp org.kde.WaylandDecoration.QAdwaitaDecorations.appdata.xml ${FLATPAK_DEST}/share/appdata",
```

Comment:
```suggestion
                "cp org.kde.WaylandDecoration.QAdwaitaDecorations.appdata.xml ${FLATPAK_DEST}/share/appdata"
```

Reply:
Yeah, I realized I probably left a comma there after I saw the json is not valid, but thanks for also suggesting the fix.